src/wp-includes/theme-templates.php

```diff
@@ -154,8 +148,15 @@ function the_block_template_skip_link() {
 			top: 5px;
 			width: auto;
 			z-index: 100000;
-		}
-	</style>
+		}';
+
+	$handle = 'wp-block-template-skip-link';
+
+	wp_register_style( $handle, false );
+	wp_add_inline_style( $handle, $skip_link_styles );
+	wp_enqueue_style( $handle );
+	?>
+
 	<?php
```

Comment:
No need for this whitespace to be output here. And after #4773 this function won't output anything.

```suggestion


```